colors/build.js

```diff
@@ -25,6 +25,13 @@ const formats = {
     'footer': '}\n',
     'ext': 'css'
   },
+  'less': {
+    'output': [`/* Firefox Colors Less Variables v${metadata.version} */
+
+`],
+    'formatter': (color, variant, value) => `@${color}-${variant}: ${value};\n`,
+    'ext': 'less'
+  },
```

Comment:
This seems like an easier change to review. 😉

The one thing I would ask for is you to generate the less file and add it in to this commit, but other than that, I'm happy!

Reply:
Indeed.

I don't have Node.js installed, and we also need the GIMP palette file generated too. Can you generate both the .less and .gpl files?

Reply:
Okay, I'll merge this, and generate those in a new commit.  Thanks!